lib/minimal_pipeline/packer.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'packer-config'
+
+class MinimalPipeline
+  # Here is an example of how to use this class to build AMIs from Packer YAML
+  #
+  # ```
+  # packer = MinimalPipeline::Packer.new('path/to/packer.yml')
+  #
+  # variables = {
+  #   'foo' => 'bar',
+  # }
+  #
+  # # Retrieve
+  # new_ami_id = packer.build_ami(variables)
+  # ```
+  class Packer
+    attr_accessor :config
+
+    # Instaniate a Packer object
+    #
+    # @param packer_config [String] Path to the JSON packer config file
+    def initialize(packer_config)
+      @config = packer_config
+    end
+
+    # Parse the newly built AMI from a given packer command output
+    #
+    # @param output [String] The command output of a packer run
+    # @return [String]
+    def get_ami_id(output)
+      return if output.nil? || output.empty?
```

Comment:
should this throw an error?

Reply:
Perhaps. I voted to leave it to the consumer of the library